### ISSUE TITLE
Docker Bazel image build fails on circle

### ISSUE THREAD

**Issue by tomheller** [dynatrace-oss/barista#1252]
# Bug Report

The docker workspace build job fails on circle CI. It seems as it is not able to install bazel or does not find it correctly. 
https://app.circleci.com/pipelines/github/dynatrace-oss/barista/7362/workflows/6a2a3b5b-29be-4a36-baad-ab3a235e5a3e/jobs/44138

```
Step 7/10 : RUN ls && bazel sync --repository_cache=/root/repocache  && bazel info
 ---> Running in a9af15ade0ca
BUILD.bazel
WORKSPACE
angular.json
libs
nx.json
package-lock.json
package.json
tools
tsconfig.json

ERROR: The project you're trying to build requires Bazel 3.2.0 (specified in /root/barista/.bazelversion), but it wasn't found in /usr/bin.
 
You can install the required Bazel version via apt:
  sudo apt update && sudo apt install bazel-3.2.0
```

Issue is locally reproduceable and fails with the same problem. 
A quick `ls /usr/bin` in the docker file shows that bazel is installed, so I presume there is a version mismatch or something? 
```sh
...
base32
base64
basename
bashbug
bazel
bazel-real
c++
c++filt
c89
... 
```

**Comment by lukasholzer** [comment]
@tomheller thx for assigning will take a closer look

**Comment by furtnerl** [comment]
Will ultimately be fixed by #1266
